packages/components/src/calendar/date-picker.js

```diff
@@ -79,7 +79,7 @@ class DatePicker extends Component {
 		return (
 			<Dropdown
 				position="bottom center"
-				focusOnMount={ false }
+				focusOnMount="container"
```

Comment:
We should add a changelog entry to the [components changelog](https://github.com/woocommerce/woocommerce-admin/blob/master/packages/components/CHANGELOG.md) with this fix too